Package/EasyNetQ.DI.Autofac/EasyNetQ.DI.Autofac.nuspec

```diff
@@ -17,7 +17,7 @@
     <copyright>Copyright Mike Hadlow 2013</copyright>
     <tags>RabbitMQ Messaging AMQP REST API Ninject</tags>
     <dependencies>
-      <dependency id="EasyNetQ" version="0.35.1.0" />
+      <dependency id="EasyNetQ" version="0.40.0.0" />
```

Comment:
Did you meant 0.40.1.0? I think your version with the changed ServiceProvider is `// 0.40.1.0 Fix concurrent bugs in DefaultServiceProvider`